src/native/fs/fs_napi.cpp

```diff
@@ -86,11 +86,13 @@ struct FSWorker : public Napi::AsyncWorker
         , _uid(ThreadScope::orig_uid)
         , _gid(ThreadScope::orig_gid)
         , _errno(0)
+        , _warn_threshold_ms(100)
```

Comment:
I think the default should be to not warn if not provided from options
```suggestion
        , _warn_threshold_ms(0)
```

Reply:
👍